neurom/features/neuritefunc.py

```diff
@@ -316,8 +333,22 @@ def bifurcation_partitions(neurites, neurite_type=None):
                              neurite_filter=is_type(neurite_type)))
 
 
-def partition_asymmetries(neurites, neurite_type=None):
-    '''Partition asymmetry at bifurcation points of a collection of neurites'''
+def partition_asymmetries(neurites, neurite_type=None, variant=None):
```

Comment:
I don't think default variant should be None

Reply:
Shall I put a different name for each variant? I was thinking not to call the original definition by a variant name, but I can put variant='original' if you prefer.